### PR TITLE
Added gl3w

### PR DESCRIPTION
**Pull request by dimitriye98**

### CODE REVIEW

File: gl3w.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'formula'
```
Comment (bfontaine) [inline gl3w.rb]: You can remove this line.

File: gl3w.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+class Gl3w < Formula
+  homepage 'https://github.com/skaslev/gl3w/'
```
Comment (bfontaine) [inline gl3w.rb]: Please use double-quoted strings everywhere.

**Comment by dimitriye98** [comment]
Well, the test bot failed in building python for the dependency. I don't think there's actually anything I can do to fix that, the issue shouldn't be with my formula after all.

**Comment by dimitriye98** [comment]
Could anyone help in fixing it so it doesn't error while building python for --dependencies-only?

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: gl3w.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class Gl3w < Formula
+  desc "Gl3w is a simple OpenGL core profile"
+  homepage "https://github.com/skaslev/gl3w/"
+  head "https://github.com/skaslev/gl3w.git"
+
+  depends_on "python" => :build
+
+  def install
+    working_dir = Dir.pwd
```
Comment (DomT4) [inline gl3w.rb]: What are you trying to achieve here? The `working_dir` will be `buildpath` - Don't need a new method for that.

File: gl3w.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
+class Gl3w < Formula
+  desc "Gl3w is a simple OpenGL core profile"
+  homepage "https://github.com/skaslev/gl3w/"
+  head "https://github.com/skaslev/gl3w.git"
+
+  depends_on "python" => :build
```
Comment (DomT4) [inline gl3w.rb]: `depends_on :python if MacOS.version <= :snow_leopard` if runtime, `depends_on :python => :build if MacOS.version <= :snow_leopard` if build.

File: gl3w.rb (hunk 3 above)
Comment (dimitriye98) [inline gl3w.rb]: The var working_dir is saved outside the chdir; that is, I'm running python in the chdir block, but referencing a file in the old working dir from before the chdir.

**Pull request closed without merging**